src/main/java/catan/services/util/play/MainStageUtil.java

```diff
@@ -36,6 +37,39 @@ public void updateNextMove(GameBean game) {
         game.setCurrentMove(nextMoveNumber);
     }
 
+    public void takeResources(Resources usersResources, GameUserActionCode action) {
+        switch (action) {
```

Comment:
call 'takeResourceFromPlayer' directly where it is needed instead of this switch-case, as we already do switch-case- check in 'processAction' method